src/WeekComponent/WeekComponent.css

```diff
@@ -6,76 +6,99 @@
 .description-week-day {
   height: 26px;
   align-self: center;
-  padding: 10px;
+  padding: 15px 178px 24px 0px;
   letter-spacing: 1.19px;
-  font-family: "San Francisco";
+  font-family: 'arial';
+  font-weight: bold;
 }
 
 .pause-start-button {
-  width: 159px;
-  height: 52px;
+  width: 336px;
+  height: 60px;
   border-radius: 25px;
   border: solid 6px rgb(233, 225, 247);
   background-color:  rgb(156, 115, 219);
   cursor: pointer;
   color: white;
   align-self: center;
   margin-top: 10px;
+  font-family: sans-serif;
+  font-size: 18px;
+  font-weight: bold;
 }
 
-.reset-button {
-  width: 159px;
+.reset-switch-button {
+  width: 336px;
   height: 52px;
   border-radius: 25px;
-  border: solid 6px  rgb(192, 164, 233);
-  background-color: rgb(233, 225, 247);
+  border: solid 6px  rgb(171, 180, 190);
+  background-color: rgb(171, 180, 190);
   cursor: pointer;
-  color: black;
+  color: white;
   align-self: center;
   margin-top: 10px;
-}
+  font-family: sans-serif;
+  font-size: 18px;
+  font-weight: bold;
+ }
 
-.counting-seconds{
+.counting-seconds-data{
   display: flex;
   flex-direction: column;
-  justify-content: flex-end;
-  position: relative;
-  padding: 3px 3px 3px 3px;
-}
+  padding: 3px 3px 3px 0px;
```

Comment:
Too small to matter, better to remove.

In general, better to remove unnecessary divs, align all elements to the grid, and use CSS features to ensure consistency.